Sources/AppStoreConnectCLI/Commands/TestFlight/Sync/TestFlightPullCommand.swift

```diff
@@ -34,7 +34,8 @@ struct TestFlightPullCommand: CommonParsableCommand {
         let testers = try service.listBetaTesters(filterIdentifiers: identifiers, limit: 200)
         let groups = try service.listBetaGroups(filterIdentifiers: identifiers)
 
-        try FileSystem.writeConfiguration(apps: apps, testers: testers, groups: groups, to: outputPath)
+        let testflightProgram = TestflightProgram(apps: apps, testers: testers, groups: groups)
```

Comment:
Which one do we want to use, Test`F`light  Test`f`light? 🤔

Reply:
I guess we should just use `TestFlight` everywhere since it's the brand capitalisation